### PR TITLE
adapt to AbstractJettyBundleActivator changes

### PR DESCRIPTION
**Pull request by bbaldino**
See https://github.com/jitsi/jicoco/pull/99

### CODE REVIEW

File: src/main/java/org/jitsi/jicofo/auth/AuthBundleActivator.java [hunk 1]
```diff
@@ -109,31 +109,7 @@
...
-     * {@inheritDoc}
-     */
-    @Override
-    protected int getDefaultTlsPort()
-    {
-        // The idea of overriding Videobridge's default is probably an attempt
-        // to have Videobridge and Jicofo running on the same machine with their
-        // defaults and without them clashing.
-        return 8843;
+        super(AUTH_PNAME, "jicofo.apis.auth");
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/auth/AuthBundleActivator.java:112]: Nooo, now's our chance to fix this :)

Comment (bgrozev) [reply src/main/java/org/jitsi/jicofo/auth/AuthBundleActivator.java:112]: That's what configures the http interface where stats are available, etc...

Comment (bbaldino) [reply src/main/java/org/jitsi/jicofo/auth/AuthBundleActivator.java:112]: What do you mean?

**Comment by bbaldino** [comment]
Test failures are known flaky torture tests

**Pull request merged**